File: pset6/mario/less/mario.py
from cs50 import get_int

# Check for valid input
while True:
    height = get_int("Height:")
    if height >= 1 and height <= 8:
        break
i = 0
while i < height:
    x = height - 1
    a = 0
    while x > i:
        # Print the required number of spaces
        print(" ", end="")
        x = x - 1
    while a <= i:
        # Print the required number of blocks
        print("#", end="")
        a = a + 1
    # Line breaks
    print()
    i = i + 1

File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int get_positive_int(void);

int main(void)
{
    int height = get_positive_int();
    for (int i = 1; i <= height; i++)
    {
        for (int k = height; k > i; k--)
        {
            // print spaces
            printf(" ");
        }
        for (int j = height; j > height - i; j--)
        {
            // print hashes
            printf("#");
        }
        printf("  ");
        for (int l = height; l > height - i; l--)
        {
            printf("#");
        }
        // Move over next line
        printf("\n");
    }
}

int get_positive_int(void)
{

    int n;
    do
    {
        n = get_int("Height\n");
    }
    while (n < 1 || n > 8);
    return n; // Keep asking for input if not in range 1-8 (inclusive)

}